### PR TITLE
French translation and locale

### PR DESCRIPTION
**Pull request by AzariasB**

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        ferial: 'Pâques %{weekday}'
+      sunday: '%{week} Dimanche de Pâques'
+      ferial: '%{weekday}, %{week} semaine de Pâques'
+  rank:
+    '1_1': Triduum Pascal
+    '1_2': Jours liturgiques primaires
+    '1_3': Solennités du Calendrier Général
+    '1_4': Solennités appropriées
+    '2_5': Fête du Seigneur du Calendrier Général
+    '2_6': Dimanche désavantagés
```
Comment (igneus) [inline config/locales/fr.yml]: While my French skills are really poor, I'm slightly suspicious concerning the translation of "Unprivileged Sundays" as "Dimanche désadvantagés". The full paragraph in the Table of Liturgical Days which I shortenned to "Unprivileged Sundays" reads "Sundays of the Christmas season and Sundays in Ordinary Time." The point is, these Sundays are in no way "disadvantaged", they are just "simple Sundays", without an additional privileged status. Most of the Sundays during the liturgical year belong to this category. Is this what the French adjective "désadvantagé" means, or is there a better word to express the idea?

Comment (AzariasB) [inline config/locales/fr.yml]: Oh ok, I see what you meant now by "Unprivileged Sundays", but I'm not sure how to translate this in french ... unless using once again "Ordinary time" but would it be correct ?

Comment (igneus) [inline config/locales/fr.yml]: No, "Ordinary time" alone wouldn't be correct. Then, what about not shortening it at all and using the full formulation from the Table of Liturgical Days instead? Maybe I should later do the same in the other locales as well. The shortcut "Unprivileged Sundays" seems to be more misleading than helpful.

The full description of the rank would be ["Les dimanches du temps de Noël et les dimanches per annum."](https://www.ceremoniaire.net/depuis1969/missel_2002/nual_2.html#tp)

Comment (AzariasB) [inline config/locales/fr.yml]: Ok, I will update to `Les dimanches du temps de Noël et les dimanches per annum`. Seems that there is no french word for this, and that we have to borrow from the latin to express what it is :neutral_face:

Comment (igneus) [inline config/locales/fr.yml]: Never more relevant, but the theologian in me can't resist:

> Seems that there is no french word for this

That's not the case. There is the French term "dimanche du temps ordinaire". Occurrence of an untraslated Latin term "dimanches *per annum*" in the document is rather a matter of theological jargon then anything else.

**Comment by igneus** [comment]
Merged. Thank you very much.

**Pull request merged**